boilerplate/_lib/ensure.sh

```diff
@@ -3,17 +3,27 @@ set -euo pipefail
 
 GOLANGCI_LINT_VERSION="1.30.0"
 DEPENDENCY=${1:-}
+GOOS=${GOOS:-linux}
```

Comment:
You should really get the host OS via `go env GOOS` instead of checking the variable as it will fail to execute if the downloaded file doesn't match.